packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.jsx

```diff
@@ -84,7 +84,15 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
 
     const list_with_category = () => {
         const contract_type_category_icon = getContractTypeCategoryIcons();
-        const order_arr = ['Accumulators', 'Multipliers', 'Vanillas', 'Ups & Downs', 'Highs & Lows', 'Digits'];
+        const order_arr = [
+            'Accumulators',
+            'Vanillas',
+            'Turbos',
```

Comment:
I'm adding Turbos here in order to maintain the correct order after Turbos are released in a separate PR soon.